app/js/views/reminders/reminders-list.jsx

```diff
@@ -13,12 +14,21 @@ export default class RemindersList extends React.Component {
     };
 
     this.server = props.server;
+    this.refreshReminders = props.refreshReminders;
+
+    this.dialog = null;
```

Comment:
Should we rename to `editDialog` just to avoid clash with any future extra dialogs in the list?